config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "cred_git_name": "source-git-creds",
+  "cred_git_username": "GITHUB_USER",
+  "cred_git_token": "GITHUB_TOKEN",
+  "cred_maven_name": "maven-creds"
```

Comment:
What needs to be provided in cred_maven_name ? 
As far as I know we only need GIt user and token